src/zherebko/index.js

```diff
@@ -24,12 +24,25 @@ export default function() {
     let maxIndex = 0;
     dag.eachLinks(({ data }) => {
       minIndex = Math.min(minIndex, data.index);
-      maxIndex = Math.max(maxIndex, data.index);
+      maxIndex = Math.max(maxIndex, data.index > 0 ? data.index : 1);
```

Comment:
This looks like it's meant to stop a divide by zero error in the case where its a line graph. However, I think the better way to handle this is just an if after the `eachLinks` like `if (minIndex == maxIndex) { minIndex = -1; maxIndex = 1;}`